RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Interface.swift

```diff
@@ -231,14 +242,6 @@ public struct SigningFactor: Sendable, Hashable, Identifiable {
 	}
 }
 
-extension DeviceFactorSource {
```

Comment:
moved to `Stage2MigrateToSargon+DeviceFactorSource`: https://github.com/radixdlt/babylon-wallet-ios/blob/sargon_profile_integration_stage_1_batch_1/RadixWallet/ProfileSargonMigration/Stage2/Atoms/Stage2MigrateToSargon%2BDeviceFactorSource.swift#L5